CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.12.0rc1-0.31b0...HEAD)
 
+- Adding escape call to fix [auto-instrumentation not producing spans on Windows](https://github.com/open-telemetry/opentelemetry-python/issues/2703).
+  ([#1098](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1098))
```

Comment:
```suggestion
  ([#1100](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1100))
```

Reply:
@jeremydvoss Either use this commit suggestion UI or commit from your machine to update the PR number. In the end we squash all your commits before merging so the multiple commits is not a problem if you are wondering about that.

Reply:
I was wondering about that. Do you mean that the commits are squashed automatically? I didn't know squashing commits was possible once they are pushed to a public remote.

Reply:
Before we merge all your commits are squashed.